extensions/wikia/RWEPageHeader/scripts/rwe-page-header.js

```diff
@@ -80,9 +81,13 @@ require(['wikia.window', 'jquery', 'wikia.tracker'], function (window, $, tracke
 	}
 
 	$(function () {
-		addWordmarkTracking();
-		initReadDropdown();
-		moveBannerNotifications();
-		openChatWindowOnClick();
+		var experimentGroup = window.Wikia.AbTest.getGroup(experimentName);
+
+		if (experimentGroup === 'TEST_1' || experimentName === 'TEST_2') {
```

Comment:
I'd suggest making the group names more explicit in future, but let's leave it like that for now.